### PR TITLE
Implemented API Key and Updated Documentation

### PR DESCRIPTION
**Pull request by ACscooter**
Also dropped R Client API

### CODE REVIEW

File: datacommons/utils.py [hunk 1]
```diff
@@ -49,6 +51,25 @@
 # The default value to limit to
 _MAX_LIMIT = 100
 
+# Environment variables used by the package
+_ENV_VAR_API_KEY = 'DC_API_KEY'         # The API KEY
```
Comment (shifucun) [inline datacommons/utils.py]: should this be None by default?

Comment (ACscooter) [reply datacommons/utils.py]: Ah this is the environment variable name, not the value itself. I can clarify in the code

File: README.md [hunk 2]
```diff
@@ -1,34 +1,42 @@
-# Data Commons Python Client
+# Data Commons Python Client API
 
-This is a python library for accessing the Data Commons API (datacommons.org).
+This is a Python library for accessing data in the Data Commons knowledge graph.
+To get started, install this package from pip.
 
-## Licence
+    pip install git+https://github.com/google/datacommons.git@stable-1.x
```
[the remark this answers is not in the document]
Comment (ACscooter) [reply README.md]: Conferred offline

**Comment by ACscooter** [comment]
Fixed #68 

Implements #75

**Pull request merged**